pa11y_config.json

```diff
@@ -0,0 +1,61 @@
+{
```

Comment:
I'd prefer this file be moved out of the project root, and into a subdir of its own (or some other idea).